<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_caltefe</class>
 <widget class="QMainWindow" name="MainWindow_caltefe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>423</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>583</width>
    <height>99999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow_caltefe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="2" colspan="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Tempo total necessário: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QComboBox" name="comboBox_diamFuro">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QComboBox" name="comboBox_guia">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Sim</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Não</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Qtd. Furos</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Guia</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="comboBox_espessura">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Espessura dos espelhos juntos</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Espessura</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5" alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_tempoTotalDias">
      <property name="text">
       <string>----</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="4">
     <widget class="QTableWidget" name="tableWidget_duration">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::DotLine</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Descrição</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tempo (horas)</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ø do furo</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLabel" name="label_tempoTotal">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>----</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pushButton_incluir">
      <property name="text">
       <string>Incluir</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="spinBox_qtdFuros">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>337</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAtualizar_par_metros"/>
   </widget>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionAtualiza_o">
   <property name="text">
    <string>Atualização</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionAtualizar_par_metros">
   <property name="text">
    <string>Atualizar parâmetros</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
